articles/aks/troubleshooting.md

```diff
@@ -28,6 +28,37 @@ You need to [request cores](https://docs.microsoft.com/azure/azure-supportabilit
 The maximum pods-per-node setting is 30 by default if you deploy an AKS cluster in the Azure portal.
 The maximum pods-per-node setting is 110 by default if you deploy an AKS cluster in the Azure CLI. (Make sure you're using the latest version of the Azure CLI). This default setting can be changed by using the `–-max-pods` flag in the `az aks create` command.
 
+## Is it safe if the IP address ranges overlap with one another or with on-premise networks?
+
+Overlapping subnets and IP ranges could cost several days of lost productivity if issues come up because of deviation from recommended practices.
+ 
+This could help shed some light on some of the issues one could encounter as a result of such overlaps: 
+ 
+In advanced networking scenarios, the IP address plan for an AKS cluster consists of the following partitions:
+ 
+1.	virtual network
+2.	at least one subnet (for nodes, pods Kubernetes and Azure resources)
+3.	a Kubernetes service address range.
+4.	Range of ips for a docker bridge
+
+The subnet(s) in (2) are partitions from the virtual network (1) and is used to allocate addresses to nodes, pods, and internal ingress or load balancer ips. Advanced planning is necessary to ensure that it will accommodate or fit in all the nodes, pods, and all Kubernetes and Azure resources that might be provisioned in your AKS cluster.
+ 
+The service address range in (3) is where ip for services will be allocated from. The Kubernetes DNS IP address used by cluster service discovery (kube-dns) will also come from (3).
+ 
+If you do not specify a non-conflicting CIDR for (4), Docker can make this decision for you and it could end up with conflicts that cause routing problems for apps or other resources in the cluster. You should select a CIDR for the Docker bridge network address. It also has to be an address space that does not collide with the rest of the CIDRs on your networks including the partitions for (1), (2) and (3).
```

Comment:
Not correct, please check the usage of docker CIDR on the network docs, it's only used on build cases, and should only not collide with 1,2,3, but it's fine to collide with anything else.